conans/client/graph/compatibility.py

```diff
@@ -88,17 +92,38 @@ def app_compat(conanfile):
 """
 
 
+def get_default_compatibility():
```

Comment:
Remove this, it is pure boilerplate without a purpose.

Reply:
Then, I'll use only functions instead.